### PR TITLE
Add some more LBP PSP Q&A entries

### PR DESCRIPTION
**Pull request by Beyley**
These are some of the more important issues people have run into, so ive copied them from the tutorial video's description

### CODE REVIEW

File: Docs/topics/discord/faq.md [hunk 1]
```diff
@@ -31,6 +31,14 @@ It aims to be smart, communicating with the server for proper patching instructi
...
 
-## Q: I followed the PSP guide, but my game wont connect!
+## Q: I followed the PSP guide, but my game wont connect, how do I fix it?
 
-Make sure to update your game to the latest version v2.0.5, as that is the only version we support. LBP PSP Alpha, or a non-updated version will not work. If you still cannot connect, ask for help in <#1049223665998372947>.
+Make sure to update your game to the latest version v2.0.5, as that is the only version we support. LBP PSP Alpha, or a non-updated version will not work. Also make sure to update your PSP to a CFW based on 6.61. If you still cannot connect, ask for help in <#1049223665998372947>.
+
+## Q: My PSP can't connect to my WiFi network, is there any way to still connect?
+
+You can setup a 2.4G hotspot on your phone or computer with no password, then use that on your PSP.
```
Comment (jvyden) [inline Docs/topics/discord/faq.md]:
```suggestion
The PSP does not support modern WPA/WPA2 networks.

You can setup a 2.4Ghz hotspot on your phone or computer with no password, and then use that on your PSP. You can also set up a WEP network if you want to use a password, although you should still use caution as it's still not as secure as WPA2.
```

File: Docs/topics/discord/faq.md [hunk 2]
```diff
@@ -31,6 +31,14 @@ It aims to be smart, communicating with the server for proper patching instructi
...
+
+## Q: The Vita web browser crashes when trying to go to a website, how do I fix this?
+
+Disable the reScaler plugin, it conflicts with the PSP web browser.
```
Comment (jvyden) [inline Docs/topics/discord/faq.md]:
```suggestion
You should try disabling the `reScaler` plugin, as it conflicts with the PSP web browser.
```

File: Docs/topics/discord/faq.md [hunk 3]
```diff
@@ -31,6 +31,14 @@ It aims to be smart, communicating with the server for proper patching instructi
...
+## Q: I followed the PSP guide, but my game wont connect, how do I fix it?
 
-Make sure to update your game to the latest version v2.0.5, as that is the only version we support. LBP PSP Alpha, or a non-updated version will not work. If you still cannot connect, ask for help in <#1049223665998372947>.
+Make sure to update your game to the latest version v2.0.5, as that is the only version we support. LBP PSP Alpha, or a non-updated version will not work. Also make sure to update your PSP to a CFW based on 6.61. If you still cannot connect, ask for help in <#1049223665998372947>.
+
+## Q: My PSP can't connect to my WiFi network, is there any way to still connect?
+
+You can setup a 2.4G hotspot on your phone or computer with no password, then use that on your PSP.
+
+## Q: The Vita web browser crashes when trying to go to a website, how do I fix this?
```
Comment (jvyden) [inline Docs/topics/discord/faq.md]: Do you mean the browser in LiveArea or the PSP browser running under adrenaline?

File: Docs/topics/discord/faq.md [hunk 4]
```diff
@@ -31,6 +31,14 @@ It aims to be smart, communicating with the server for proper patching instructi
 
 Not at the moment. PPSSPP support is in the works, but is currently not possible.
 
-## Q: I followed the PSP guide, but my game wont connect!
+## Q: I followed the PSP guide, but my game wont connect, how do I fix it?
```
Comment (jvyden) [inline Docs/topics/discord/faq.md]: Old way reads better tbh

Thinking of dropping the `Q: ` portion

File: Docs/topics/discord/faq.md [hunk 5]
```diff
@@ -31,6 +31,14 @@ It aims to be smart, communicating with the server for proper patching instructi
...
 
-Make sure to update your game to the latest version v2.0.5, as that is the only version we support. LBP PSP Alpha, or a non-updated version will not work. If you still cannot connect, ask for help in <#1049223665998372947>.
+Make sure to update your game to the latest version v2.0.5, as that is the only version we support. LBP PSP Alpha, or a non-updated version will not work. Also make sure to update your PSP to a CFW based on 6.61. If you still cannot connect, ask for help in <#1049223665998372947>.
```
Comment (jvyden) [inline Docs/topics/discord/faq.md]:
```suggestion
Make sure to update your game to the latest version v2.0.5, as that is the only version we support. LBP PSP Alpha, or a non-updated version will not work. Additionally, PSN will only work on PSPs running a CFW based on the latest firmware (6.61 is the latest official version as of writing) so make sure your custom firmware is up to date.

If you still cannot connect, ask for help in <#1049223665998372947>.
```

**Review by jvyden — CHANGES_REQUESTED** [review]

File: Docs/topics/discord/faq.md (hunk 3 above)
Comment (Beyley) [reply Docs/topics/discord/faq.md]: The PSP browser under adrenaline, should have specified

File: Docs/topics/discord/faq.md (hunk 4 above)
Comment (Beyley) [reply Docs/topics/discord/faq.md]: For now i'll just revert that one line, revamps to the whole file should probably be done in a different patchest

**Review by jvyden — APPROVED** [review]

**Pull request merged**